### PR TITLE
Add BlenderBot small tokenizer to the init

### PR DESCRIPTION
**Pull request by LysandreJik**
This class was forgotten, adding it to the init and the documentation.

### CODE REVIEW

File: src/transformers/__init__.py [hunk 1]
```diff
@@ -360,6 +360,7 @@
     _import_structure["models.roformer"].append("RoFormerTokenizerFast")
     _import_structure["models.clip"].append("CLIPTokenizerFast")
     _import_structure["models.convbert"].append("ConvBertTokenizerFast")
+    _import_structure["models.blenderbot_small"].append("BlenderbotTokenizerSmallFast")
```
Comment (patil-suraj) [inline src/transformers/__init__.py:363]:
```suggestion
    _import_structure["models.blenderbot_small"].append("BlenderbotSmallTokenizerFast")
```

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for adding this !

**Review by patrickvonplaten — APPROVED** [review]

**Review by sgugger — APPROVED** [review]
LGTM!

**Comment by LysandreJik** [comment]
Actually this tokenizer seems a bit broken:
```py
tokenizer = AutoTokenizer.from_pretrained("facebook/blenderbot_small-90M")
```
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/lysandre/Workspaces/Python/transformers/src/transformers/models/auto/tokenization_auto.py", line 469, in from_pretrained
    return tokenizer_class_fast.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "/home/lysandre/Workspaces/Python/transformers/src/transformers/tokenization_utils_base.py", line 1741, in from_pretrained
    return cls._from_pretrained(
  File "/home/lysandre/Workspaces/Python/transformers/src/transformers/tokenization_utils_base.py", line 1858, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)
  File "/home/lysandre/Workspaces/Python/transformers/src/transformers/models/blenderbot_small/tokenization_blenderbot_small_fast.py", line 76, in __init__
    ByteLevelBPETokenizer(
  File "/home/lysandre/transformers/.env/lib/python3.8/site-packages/tokenizers/implementations/byte_level_bpe.py", line 36, in __init__
    BPE(
Exception: Error while initializing BPE: Token `_</w>` out of vocabular
```

**Comment by LysandreJik** [comment]
Which only appears after fixing with 6d90d5a3213b812f3d54d6142751a275b59343b1

**Pull request merged**